pkg/cmd/server/crypto/crypto.go

```diff
@@ -147,8 +155,11 @@ func GetCA(certFile, keyFile, serialFile string) (*CA, error) {
 	}, nil
 }
 
-func MakeCA(certFile, keyFile, serialFile, name string) (*CA, error) {
+func MakeCA(certFile, keyFile, serialFile, name string, out io.Writer) (*CA, error) {
```

Comment:
> Is this going to be used for anything but creating the openshift CA?

No reason why it couldn't.  Things that come to mind right away:
 1. Create a CA for generated certs to secure service communication in a namespace.
 1. Create a CA to generate a short lived, signed key for authentication.
 1. Create a CA to sign a short lived scoping of permissions in a token.

In response to 2, see the idea of returning a bool from the `Ensure` method.

In response to 3, unsuppressable output from a utility method always causes a problem eventually.